app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -270,6 +270,34 @@ function handleKeyUp(event) {
   handleSizeChange(event);
 }
 
+function handleKeyDown(event) {
+    // If Ctrl+Enter OR Command+Enter is pressed
+    if((event.ctrlKey || event.metaKey) && event.keyCode === 13)
+    {
```

Comment:
What do you think about moving this bracket to the line above? This would make the code style more consistent to the style in the whole file. Also this is what [airbnb](https://github.com/airbnb/javascript) recommends and it's the style used in this project.